src/main.rs

```diff
@@ -35,8 +34,17 @@ enum TargetGen {
         )]
         output_dir: PathBuf,
     },
+    /// Generate target description from ARM CMSIS-Packs
```

Comment:
The description is currently the same as for the `pack` sub command, which is quite confusing in the CLI output.